apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -115,16 +115,23 @@ function DescriptionSection(props: DescriptionProps) {
   );
 }
 
-type CtaProps = Pick<Twc2SupportDetail, 'ctaBlurb' | 'ctaButtonText'>;
+type CtaProps = Pick<
+  Twc2SupportDetail,
+  'ctaBlurb' | 'ctaButtonText' | 'ctaLink'
+>;
 
 function CtaSection(props: CtaProps) {
-  const { ctaBlurb, ctaButtonText } = props;
+  const { ctaBlurb, ctaButtonText, ctaLink } = props;
   return (
     <Box className="support-cta-container" sx={{ borderTopColor: 'grey.200' }}>
       <div style={{ flex: 4, height: '100%' }}>{ctaBlurb}</div>
       <PrimaryButton
+        href={ctaLink}
+        target="_blank"
+        rel="noopener"
         sx={{ flex: 5, backgroundColor: 'primary.700', height: '100%' }}
       >
+        <WhatsappIcon></WhatsappIcon>
```

Comment:
@e056 I am not sure if it's possible but can we explore switching up the icon as well based on some props?

For example, for Pro Bono SG we will use a schedule (calendar) icon instead. If not then that's fine too!

I think what we can do is have a prop for `icon`, and then do a conditional within the button to choose which button to render. @glencbz any thoughts?

Reply:
Ahh okie sure, I will look into that

Reply:
I think it's okay not to worry about it for now, we can alway refactor out the commonality if we need to (we may not necessarily need it).

Reply:
Cool! Later is fine too.